lib/base-types.ts

```diff
@@ -18,6 +18,7 @@ export interface ConfigParams<T extends ShopifyRestResources = any> {
   adminApiAccessToken?: string;
   userAgentPrefix?: string;
   privateAppStorefrontAccessToken?: string;
+  usePrivateToken?: boolean;
```

Comment:
I wonder if this name is clear enough - should we call this `usePrivateStorefrontAccessToken` for extra clarity?